### ISSUE TITLE
Repaints on scroll

### ISSUE THREAD

**Issue by aydun1** [PolymerElements/paper-scroll-header-panel#41]
I've noticed that this elements "repaints on scroll" according to Chrome developer tools. On slower devices and Firefox Mobile this seems to have a slight impact on scroll performance.

More noticeably, having an element that also repaints on scroll inside a `paper-scroll-header-panel`, on mobile devices, demonstrates some lag. When you scroll the page, the contents of the inner scrollable element don't keep up.

The Polymer Docs page, which has a similar header, doesn't repaint on scroll and seems much more responsive. A good comparison is these two pages in Firefox Mobile (this browser seems to have the most trouble)

Doesn't repaint:
https://www.polymer-project.org/1.0/docs/devguide/properties.html

Repaints:
https://elements.polymer-project.org/bower_components/paper-scroll-header-panel/demo/index.html

I guess my question is if the repainting actually responsible for the relatively poor performance on some devices and browsers? And is it inevitable that it repaints? The polymer docs page doesn't, but the header also doesn't hide on scrolling..

**Comment by blasten** [comment]
**I filed an issue in Chrome:** https://code.google.com/p/chromium/issues/detail?id=514303

In Chrome (I don't know about other browsers), this element will trigger repaint on scroll if the screen density of the display is very low. (It won't in retina display) This is caused by adding the CSS property:  `overflow:auto` to any element.  I removed a `translateZ(0)` https://github.com/PolymerElements/paper-scroll-header-panel/commit/6bb3e6a7d12c61c235401316b7b1d22b58926e6b we had before to prevent repainting because it compromised other performance metrics.  

The polymer website uses the document scroll and `position:fixed` for the header. As a result, you don't see repaint rectangles when you scroll because document scroll is always GPU accelerated.

That being said, we are working on new layout elements that will use the document and `position:fixed` in addition to the current use case. Feel free to play with it: https://github.com/PolymerLabs/app-layout

cc @frankiefu
